allink_core/core/models/base_plugins.py

```diff
@@ -357,8 +351,8 @@ def get_render_queryset_for_display(self, category=None):
         queryset = self._apply_filtering_to_queryset_for_display(queryset)
 
         # apply ordering
-        # if apply_ordering:
-        queryset = self._apply_ordering_to_queryset_for_display(queryset)
+        if apply_ordering:
```

Comment:
@tuerlefl uncommenting this line causes this error:
```
UnboundLocalError at /de/
local variable 'apply_ordering' referenced before assignment
```

Can we comment it again or is there another fix?

Reply:
there is an other fix, i will create a PR

Reply:
@slavanga https://github.com/allink/allink-core/pull/172